docs/ops/detection/PriorBox_1.md

```diff
@@ -82,80 +116,36 @@
 
     * **Description**: *fixed_ratio* is an aspect ratio of a box. For example, *fixed_ratio* equal to 2.000000 means that the aspect ratio for the first box aspect ratio is 2.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
     * **Default value**: None
     * **Required**: *no*
 
 * *fixed_size*
 
     * **Description**: *fixed_size* is an initial box size (in pixels). For example, *fixed_size* equal to 15 means that the initial box size is 15.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
     * **Default value**: None
     * **Required**: *no*
 
 * *density*
 
     * **Description**: *density* is the square root of the number of boxes of each type. For example, *density* equal to 2 means that the first box generates four boxes of the same size and with the same shifted centers.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
     * **Default value**: None
     * **Required**: *no*
 
 **Inputs**:
 
-*   **1**: `output_size` - 1D tensor with two integer elements `[height, width]`. Specifies the spatial size of generated grid with boxes. Required.
+*   **1**: `output_size` - 1D tensor with two integer elements `[height, width]`. Specifies the spatial size of generated grid with boxes. **Required**.
```

Comment:
Please introduce *Types* section and refer to it from inputs/outputs section. We need to be consistent with the rest of spec.